src/mdev.rs

```diff
@@ -16,6 +16,94 @@ pub enum FormatType {
     Defined,
 }
 
+pub struct MDevSysfsData {
+    pub uuid: Uuid,
+    pub active: bool,
+    pub parent: Option<String>,
+    pub mdev_type: Option<String>,
```

Comment:
I don't think it makes sense to have an `active` field here or `Option`s for the other fields. 

```suggestion
    pub parent: String,
    pub mdev_type: String,
```

If the device is not active, the `load()` function it should simply not return a `MDevSysfsData` object. Having all of these fields be separately nullable implies that it might be valid to have an object where `active` is true, and `parent` is `None`, etc.

Reply:
See my comment on `new()`. 
`active` can be `false` and `parent `as well as m`dev_type` are than `None`.
Maybe the struct fields should not be public and only accessible via getters.